src/Database/Log/QueryLogger.php

```diff
@@ -44,10 +44,13 @@ public function __construct(array $config = [])
      */
     public function log($level, $message, array $context = [])
     {
-        Log::write(
-            'debug',
-            'connection=' . $this->getConfig('connection', '') . ' ' . (string)$context['query'],
-            ['scope' => $this->scopes() ?: ['queriesLog']]
-        );
+        $context['scope'] = $this->scopes() ?: ['queriesLog'];
+        $context['connection'] = $this->getConfig('connection');
+
+        if ($context['query'] instanceof LoggedQuery) {
+            $context = $context['query']->getContext() + $context;
+            $message = 'connection={connection} duration={took} rows={numRows} ' . $message;
```

Comment:
Does this print `connection={connection}` if the connection name context isn't set?

Reply:
Yes.

Reply:
Maybe it should be empty string.

Reply:
:+1: Added a default value for connection.